app/external_doc.rb

```diff
@@ -59,17 +59,15 @@ def call
 
         href = element["href"].strip
         uri = URI.parse(href)
-        path = uri.path
 
         next if uri.scheme || href.start_with?("#")
 
-        base = if path.start_with? "/"
-                 base_url
-               else
-                 context[:subpage_url]
-               end
-
-        element["href"] = URI.join(base, href).to_s
+        if href.start_with?("/") && context[:subpage_url].to_s.start_with?("https://github.com")
```

Comment:
The extra check here puzzles me, as we already seem to assume "external doc" == "GitHub doc" at the top of the file. If there's an edge case, could we clarify it with a test?

Reply:
That's a fair point. We _do_ only import GitHub docs, and that assumption is made in lots of places. I tacked this on as an extra fail-safe but we don't currently have a situation where we need it. I'll remove the extra check.